src/core/selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { Selectors as FluxSelectors, Store } from '@storefront/flux-capacitor';
+import Configuration from './configuration';
+import ProductTransformer from './product-transformer';
+
+namespace Selectors {
+  export const productTransformer = (state: Store.State) => {
+    const config: Partial<Configuration> = FluxSelectors.config(state);
+
+    return ProductTransformer.transformer(config.structure);
+  };
+
+  export const transformedProducts = (state: Store.State) => {
+    const config: Partial<Configuration> = FluxSelectors.config(state);
+
+    return FluxSelectors.productsWithPastPurchase(state, config.recommendations.idField)
```

Comment:
Should the config adapter be used?

Reply:
There is no idField adapter + it would be more verbose even if there was one